cmd/amplifier/server/configuration.go

```diff
@@ -10,17 +10,17 @@ import (
 )
 
 const (
-	DefaultPort          = ":50101"
-	DefaultPublicAddress = "local.appcelerator.io"
-	DefaultTimeout       = time.Minute
+	DefaultPort       = ":50101"
+	DefaultPublicHost = "127.0.0.1"
```

Comment:
gRPC is not really a well known scheme, and port guessing can be confusing for users. I'd rather set a global endpoint as "address:port" for simplicity sake.